public/index.html

```diff
@@ -20,7 +25,29 @@
 </head>
 
 <body>
-    Hello 12
+    <nav class="navbar navbar-expand-lg sticky-top navbar-dark bg-dark">
+        <a class="navbar-brand text-white px-2" style="font-weight: bold;" href="index.html">
+            <img src="logo.png" width="30" height="30" alt="" loading="lazy">
+            Work Trolly
+        </a>
+         <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarText" aria-controls="navbarText" aria-expanded="false" aria-label="Toggle navigation">
+          <span class="navbar-toggler-icon text-white"></span>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarText">
+            <ul class="navbar-nav ml-auto">
+                <li class="nav-item active pr-5 py-1">
+                    <button type="button" class="btn btn-outline-info" id="createNewTaskButton">Create New Task</button>
+                </li>
+                <li class="nav-item pr-5 py-1">
+                    <button type="button" class="btn btn-outline-info" id="allTaskButton">All Task</button>
+                </li>
+                <li class="nav-item py-1" id="login">
```

Comment:
Id is not required here

Reply:
okay i will remove id from the login button